src/main/kotlin/org/openredstone/chad/commands/fractals.kt

```diff
@@ -0,0 +1,128 @@
+package org.openredstone.chad.commands
+
+import kotlin.math.*
+import kotlin.random.Random
+import java.awt.image.BufferedImage
+import java.security.MessageDigest
+
+fun fractal(seed: String): BufferedImage {
+    val width = 2048
+    val height = 2048
+    val maxIterations = 10_000  // don't worry it will average like 5 iterations per pixel
+    val messiness = 30          // unless you change this to something like 99
+    val zoom = 3.5
+
+    val aspectRatio = width.toDouble() / height.toDouble()
+    val rng = Random(seed.sha256lowerLong())
+
+    val angle = rng.nextDouble(-3.14, 3.14)
+    val seedCoordinate = findGoodJulia(angle, messiness)
+
+    val a = rng.nextDouble(0.0, 0.2)
+    val b = rng.nextDouble(0.0, 0.2)
+    val c = rng.nextDouble(0.0, 0.2)
+
+    val img = BufferedImage(width, height, BufferedImage.TYPE_3BYTE_BGR)
+    for (y in 0 until height) {
+        for (x in 0 until ceil(width.toDouble() / 2.0).toInt()) {
+            val coX = aspectRatio * zoom * (x.toDouble() / width.toDouble() - 0.5)
+            val coY = zoom * (y.toDouble() / height.toDouble() - 0.5)
+            val coordinate = Complex(coX, coY)
+            val iterations = juliaPixel(coordinate, maxIterations, seedCoordinate)
+            val color = getColor(iterations, a, b, c)
+            img.setRGB(x, y, color)
+            img.setRGB(width - x - 1, height - y - 1, color)
+        }
+    }
+    return img
+}
+
+// Helper functions
+fun getColor(i: Double, a: Double, b: Double, c: Double): Int {
+    val red = max(sin(i * a) * 255.0, 0.0).toInt()
+    val green = max(sin(i * b) * 255.0, 0.0).toInt()
+    val blue = max(sin(i * c) * 255.0, 0.0).toInt()
+    return red + green.shl(8) + blue.shl(16)
+}
+
+fun juliaPixel(coordinate: Complex, maxIterations: Int, c: Complex): Double {
+    var z = coordinate
+    var i = 0
+    while (i < maxIterations && z.mag2() < 4.0) {
+        z = z.square() + c
+        i += 1
+    }
+    if (i < maxIterations) {
```

Comment:
A bit of a nitpick, but I would negate this condition and do an early exit instead, to reduce nesting:
```
if (i >= maxIterations) {
  return i.toDouble()
}
repeat(3) { // ...
}
// actual magic
return ...
```